gamestonk_terminal/stocks/fundamental_analysis/dcf_model.py

```diff
@@ -224,234 +224,187 @@ def others_in_sector(ticker: str, sector: str, industry: str) -> List[str]:
     industry = industry.replace("—", " - ")
     industry = industry.replace("/", " ")
 
```

Comment:
I find the wording _sister companies_ confusing. A sister company has a 'parent' company, see below.
![image](https://user-images.githubusercontent.com/46355364/149842741-46c363cd-70a0-43e0-9643-eed7cc885318.png)

Reply:
Yup I definitely used the wrong word here. All changes made.